parambokeh/__init__.py

```diff
@@ -475,3 +476,67 @@ def click_cb():
 
         outputs = [self.widget(pname) for pname in outputs]
         return widgets, outputs
+
+
+class JSONInit(param.Parameterized):
+    """
+    Callable that can be passed to Widgets.initializer to set Parameter
+    values using JSON. There are three approaches that may be used:
+    1. If the json_file argument is specified, this takes precedence.
+    2. The JSON file path can be specified via an environment variable.
+    3. The JSON can be read directly from an environment variable.
+    Here is an easy example of setting such an environment variable on
+    the commandline:
+    PARAMBOKEH_INIT='{"p1":5}' jupyter notebook
```

Comment:
Shouldn't this be PARAM_INIT, so that user code doesn't depend on whether the author chose ParamNB or ParamBokeh?

Reply:
+1

Reply:
Sure. Though this initializer is only supported by the widget systems (paramnnb/parambokeh). So how about ``PARAM_WIDGETS`` or similar?

Reply:
It is *currently* only supported by the widget systems, but there is nothing about it that is related to widgets, so we can and should add something to Param that consumes it as well.  I.e., the implementation should move to Param, and Param also needs an example of using it.  But all those things are orthogonal; for now we just need a name that doesn't make any assumptions about how the declared values will be consumed.

Reply:
I've chosen ``PARAM_JSON_INIT`` as that is the most accurate description of what it is.